nautobot/extras/migrations/0051_relationship_required_side.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2022-10-13 15:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0050_add_job_task_queues"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="relationship",
+            name="required_side",
+            field=models.CharField(blank=True, default="", max_length=12),
```

Comment:
This looks like it doesn't match with the model field definition below (notably the default value) - might need to regenerate this migration